theme/src/ts/misc.ts

```diff
@@ -102,6 +102,10 @@ function generateOnThisPage() {
 
         $("h2, h3").each(function () {
             var $el = $(this);
+            // Skip if this heading is inside a hidden element
```

Comment:
No heading in sidebar if hidden A/B heading

Reply:
There is an "on this page" sidebar. And it is showing elements that aren't visible on the page, because they are inside a hidden element. This fixes that.

Reply:
ahhh gotcha. they are showing up in the on this page side bar because they are hidden in the main content area of the page and we are just scraping the heading tags when we generate the on this page list. ok that makes sense!!